api.bs

```diff
@@ -676,18 +676,17 @@ for=PrivateAttribution>measureConversion()</a> method to find matching
 
 ### Contents ### {#impression-store-contents}
 
-The [=impression store=] must store the following information:
+The [=impression store=] must store the following information per [=impression=]:
 
-<div link-for=PrivateAttribution>
+<div dfn-for=impression link-for=PrivateAttribution>
 <pre class=simpledef>
-Filter Data: The {{PrivateAttributionConversionOptions/filterData}} value passed to <a>saveImpression()</a>.
-Impression Site: The site that called <a>saveImpression()</a>.
-Intermediary Site: The site corresponding to the script that called <a>saveImpression()</a>.
-Conversion Sites: The conversion site(s) that were passed to <a>saveImpression()</a>.
-Timestamp: The time at which <a>saveImpression()</a> was called.
-Lifetime: The number of days an [=/impression=] remains eligible for attribution,
-Lifetime: either from the call to <a>saveImpression()</a>, or a [=/user agent=]-defined limit.
-Histogram Index: The histogram index passed to <a>saveImpression()</a>.
+<dfn>Filter Data</dfn>: The {{PrivateAttributionConversionOptions/filterData}} value passed to <a>saveImpression()</a>.
```

Comment:
Do we need to make these all uppercase?

Reply:
We don't need to, but I tried to leave the formatting as is since it was presumably chosen for a reason.